index.html

```diff
@@ -58,133 +54,174 @@ <h2>Vision</h2>
-          to safeguard your organization against evolving cyber threats.
+          comprehensive cybersecurity training, seeking the vigilant oversight
+          of our Managed Security Services Security Operations Centre (SOC), or
+          aiming to streamline your governance, risk management, and compliance
+          with our Governance, Risk and Compliance solutions, we're ready to
+          provide top-notch services to meet your needs. Additionally, we offer
+          tailored Training services to equip your team with the skills
+          necessary to safeguard your organization against evolving cyber
+          threats.
         </p>
-        <button>Contact us</button>
+        <button class="contacts-btn">Contact us</button>
       </div>
       <div class="consultation">
-        <img src="" alt="" />
-        <h4>We are here 24/7</h4>
-        <h2>Need a consultation?</h2>
-        <button>Let's get started</button>
+        <div class="consultation-img">
+          <img src="assets/img/consultation2.svg" alt="" class="lady-image" />
+        </div>
+        <div class="consultation-content">
+          <h2>Need a consultation?</h2>
+          <h3>We'd love to hear from you.</h3>
+          <h3>We are here to answer your questions 24/7</h3>
+
+          <p>
+            Please feel free to reach out to us with any inquiries, concerns, or
+            collaboration opportunities. Your security journey begins with us,
+            and we're committed to safeguarding your digital landscape. Contact
+            us today, and let's fortify your defenses together.
+          </p>
+          <button class="consultation-btn">Let's get started</button>
+        </div>
       </div>
     </section>
     <footer>
-      <img src="" alt="" />DSEC
-      <ul>
-        <li><a href="#home">Home</a></li>
-        <li><a href="#explore">Explore</a></li>
-        <li><a href="#products">Products</a></li>
-        <li><a href="#contact">Contact us</a></li>
-      </ul>
-      <p>
-        P.O. BOX 44848 – 00100 <br />
-        Nairobi, Kenya <br />
-        hello@dsec.co.ke <br />
-        +254 746 191 652
-      </p>
-      <img src="" alt="" />
-      <img src="" alt="" />
-      <img src="" alt="" />
-      <img src="" alt="" />
-      <p>DSEC Copyright © 2025. All Rights Reserved.</p>
+      <div class="footer-logo">
+        <img src="assets/img/footer-logo.svg" alt="DSEC" class="dsec-footer" />
+        <div class="footer-links">
+          <div class="links">
+            <ul>
+              <li><a href="#home">Home</a></li>
+              <li><a href="#explore">Explore</a></li>
+              <li><a href="#products">Products</a></li>
+              <li><a href="#contact">Contact Us</a></li>
+            </ul>
+          </div>
+          <div class="address">
+            <ul>
+              <li>P.O. BOX 44848 – 00100</li>
+              <li>Nairobi, Kenya</li>
+              <li><a href="#email">hello@dsec.co.ke</a></li>
```

Comment:
When this link is clicked, it should direct the user to say gmail, and automatically start a mail body. Research on this.